// zlibjs/cfg/jsRes.js
var js = js || {};
js.zlibjs = {
    adler32_js : '[%zlibjs%]src/adler32.js',
    bitstream_js : '[%zlibjs%]src/bitstream.js',
    crc32_js : '[%zlibjs%]src/crc32.js',
    deflate_js : '[%zlibjs%]src/deflate.js',
    export_object_js : '[%zlibjs%]src/export_object.js',
    exports_js : '[%zlibjs%]src/exports.js',
    externs_js : '[%zlibjs%]src/externs.js',
    gunzip_js : '[%zlibjs%]src/gunzip.js',
    gunzip_member_js : '[%zlibjs%]src/gunzip_member.js',
    gzip_js : '[%zlibjs%]src/gzip.js',
    heap_js : '[%zlibjs%]src/heap.js',
    huffman_js : '[%zlibjs%]src/huffman.js',
    hybrid_js : '[%zlibjs%]src/hybrid.js',
    inflate_js : '[%zlibjs%]src/inflate.js',
    inflate_stream_js : '[%zlibjs%]src/inflate_stream.js',
    node_zlib_js : '[%zlibjs%]src/node-zlib.js',
    rawdeflate_js : '[%zlibjs%]src/rawdeflate.js',
    rawinflate_js : '[%zlibjs%]src/rawinflate.js',
    rawinflate_stream_js : '[%zlibjs%]src/rawinflate_stream.js',
    unzip_js : '[%zlibjs%]src/unzip.js',
    use_js : '[%zlibjs%]src/use.js',
    util_js : '[%zlibjs%]src/util.js',
    zip_js : '[%zlibjs%]src/zip.js',
    zlib_js : '[%zlibjs%]src/zlib.js',
    browser_codepath_test_js : '[%zlibjs%]test/browser-codepath-test.js',
    browser_gunzip_test_js : '[%zlibjs%]test/browser-gunzip-test.js',
    browser_gzip_test_js : '[%zlibjs%]test/browser-gzip-test.js',
    browser_inflate_test_js : '[%zlibjs%]test/browser-inflate-test.js',
    browser_raw_test_js : '[%zlibjs%]test/browser-raw-test.js',
    browser_test_js : '[%zlibjs%]test/browser-test.js',
    browser_unzip_test_js : '[%zlibjs%]test/browser-unzip-test.js',
    browser_zip_test_js : '[%zlibjs%]test/browser-zip-test.js',
    browser_zlib_test_js : '[%zlibjs%]test/browser-zlib-test.js',
    buster_js : '[%zlibjs%]test/buster.js',
    make_deflated_testdata_js : '[%zlibjs%]test/make_deflated_testdata.js',
    node_test_js : '[%zlibjs%]test/node-test.js',
    plain_js : '[%zlibjs%]test/plain.js',
    use_typedarray_js : '[%zlibjs%]test/use_typedarray.js'
};
